src/Language/PureScript/Linter/Exhaustive.hs

```diff
@@ -291,42 +291,19 @@ checkExhaustive ss env mn numArgs cas expr = makeResult . first ordNub $ foldl'
 --
 checkExhaustiveExpr
   :: forall m
-   . MonadWriter MultipleErrors m
+   . (MonadError MultipleErrors m, MonadWriter MultipleErrors m)
    => SourceSpan
    -> Environment
    -> ModuleName
    -> Expr
    -> m Expr
-checkExhaustiveExpr initSS env mn = onExpr initSS
+checkExhaustiveExpr initSS env mn = onExpr'
```

Comment:
If we keep `initSS` as it is currently, it should probably be renamed to just `ss`.